src/vs/workbench/services/contextmenu/electron-sandbox/contextmenuService.ts

```diff
@@ -244,7 +244,9 @@ class NativeContextMenuService extends Disposable implements IContextMenuService
 	}
 
 	private async runAction(actionRunner: IActionRunner, actionToRun: IAction, delegate: IContextMenuDelegate, event: IContextMenuEvent): Promise<void> {
-		this.telemetryService.publicLog2<WorkbenchActionExecutedEvent, WorkbenchActionExecutedClassification>('workbenchActionExecuted', { id: actionToRun.id, from: 'contextMenu' });
+		if (!delegate.skipTelemetry) {
```

Comment:
@sbatten fyi missing adoption?

Reply:
Yea, I didn't realize we did it on native as well for some reason.